internal/tools/looker/lookerhealthanalyze/lookerhealthanalyze.go

```diff
@@ -0,0 +1,578 @@
+		pName := *p.Name
+		pID := *p.Id
+		logger.InfoContext(ctx, fmt.Sprintf("Analyzing project: %s", pName))
+
+		projectFiles, err := t.SdkClient.AllProjectFiles(pID, "", nil)
+		if err != nil {
+			return nil, fmt.Errorf("error fetching files for project %s: %w", pName, err)
+		}
+
+		modelCount := 0
+		viewFileCount := 0
+		for _, f := range projectFiles {
+			if f.Type != nil {
+				if *f.Type == "model" {
+					modelCount++
+				}
+				if *f.Type == "view" {
+					viewFileCount++
+				}
+			}
+		}
+
+		gitConnectionStatus := "OK"
+		if p.GitRemoteUrl == nil {
+			gitConnectionStatus = "No repo found"
+		} else if strings.Contains(*p.GitRemoteUrl, "/bare_models/") {
+			gitConnectionStatus = "Bare repo, no tests required"
+		}
+
+		results = append(results, map[string]interface{}{
+			"Project":             pName,
+			"# Models":            modelCount,
+			"# View Files":        viewFileCount,
+			"Git Connection Status": gitConnectionStatus,
+			"PR Mode":             string(*p.PullRequestMode),
+			"Is Validation Required": *p.ValidationRequired,
+		})
+	}
+	return results, nil
+}
+
+func (t *analyzeTool) models(ctx context.Context, project, model string) ([]map[string]interface{}, error) {
+	logger, err := util.LoggerFromContext(ctx)
+	if err != nil {
+		return nil, fmt.Errorf("unable to get logger from ctx: %s", err)
+	}
+	logger.InfoContext(ctx, "Analyzing models...")
+
+	usedModels, err := t.getUsedModels(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	lookmlModels, err := t.SdkClient.AllLookmlModels(v4.RequestAllLookmlModels{}, nil)
+	if err != nil {
+		return nil, fmt.Errorf("error fetching LookML models: %w", err)
+	}
+
+	var results []map[string]interface{}
+	for _, m := range lookmlModels {
+		if (project == "" || (m.ProjectName != nil && *m.ProjectName == project)) &&
+			(model == "" || (m.Name != nil && *m.Name == model)) {
+			
+			queryCount := 0
+			if qc, ok := usedModels[*m.Name]; ok {
+				queryCount = qc
+			}
+
+			exploreCount := 0
+			if m.Explores != nil {
+				exploreCount = len(*m.Explores)
+			}
+
+			results = append(results, map[string]interface{}{
+				"Project":          *m.ProjectName,
+				"Model":            *m.Name,
+				"# Explores":       exploreCount,
+				"Query Count":      queryCount,
+			})
+		}
+	}
+	return results, nil
+}
+
+func (t *analyzeTool) getUsedModels(ctx context.Context) (map[string]int, error) {
+	limit := "5000"
+	query := v4.WriteQuery{
+		Model:  "system__activity",
+		View:   "history",
+		Fields: &[]string{"history.query_run_count", "query.model"},
+		Filters: &map[string]any{
+			"history.created_date":      fmt.Sprintf("%d days", t.timeframe),
+			"query.model":               "-system__activity, -i__looker",
+			"history.query_run_count":   fmt.Sprintf(">%d", t.minQueries-1),
+			"user.dev_branch_name":      "NULL",
+		},
+		Limit: &limit,
+	}
+	raw, err := t.SdkClient.RunInlineQuery(v4.RequestRunInlineQuery{
+		Body:         query,
+		ResultFormat: "json",
+	}, nil)
```

Comment:
If you look at some of the other query calls I updated the query logic to use an alternate endpoint that is tracked. Can you do the same for yours?

Reply:
Just updated all the runinline query calls to use the alternate endpoint if supported and resort back to the original if not